site/en/docs/extensions/reference/action/index.md

```diff
@@ -156,14 +153,14 @@ chrome.action.setBadgeText(
 );
 ```
 
-If the `tabId` property is omitted, the setting is treated as a global setting. Tab-specific
-settings take priority over any global settings.
+If the `tabId` property is left out, the setting is treated as a global setting. Tab-specific
+settings take priority over global settings.
 
-## Enabled state
+### Enabled state
 
-By default, toolbar actions are enabled (clickable) on every tab. You can control this using the
+By default, toolbar actions are clickable on every tab. You can control this using the
```

Comment:
```suggestion
By default, toolbar actions are enabled (clickable) on every tab. You can control this using the
```

Reply:
I think you should change this back. 'Clickable' is a clarifying remark for a word that reflects that methods that are used.